src/main/java/ee/tuleva/onboarding/user/response/UserResponse.java

```diff
@@ -16,22 +17,34 @@
     private String personalCode;
     private String firstName;
     private String lastName;
+    private String email;
+    private String phoneNumber;
 
     public int getAge() {
-        return PersonalCode.getAge(this.getPersonalCode());
+        return PersonalCode.getAge(personalCode);
     }
 
     public static UserResponse fromAuthenticatedPerson(AuthenticatedPerson authenticatedPerson) {
-        return UserResponse.builder()
+        return builder()
                 .id(authenticatedPerson.getUser().getId())
```

Comment:
should write TODO here to add phone number/email OR remove this method and replace usage with new method all together